packages/lesswrong/server/voteServer.ts

```diff
@@ -223,11 +223,11 @@ export const performVoteServer = async ({ documentId, document, voteType, extend
   const collectionVoteType = `${collectionName.toLowerCase()}.${voteType}`
 
   if (!user) throw new Error("Error casting vote: Not logged in.");
-
+  
```

Comment:
nit: remove extra newline

Reply:
I'm confused, there's no extra space compared to Master